tools/package-tools.sh

```diff
@@ -59,6 +125,14 @@ if [ ! -e obot-tools/venv ]; then
 fi
 
 source obot-tools/venv/bin/activate
-
 find obot-tools -name requirements.txt -exec cat {} \; -exec echo \; | sort -u > requirements.txt
 uv pip install -r requirements.txt
+
+cd obot-tools
+cat <<EOF > .envrc.tools
+export GPTSCRIPT_SYSTEM_TOOLS_DIR=/obot-tools/
+export GPTSCRIPT_TOOL_REMAP="$(IFS=','; echo "${REGISTRY_REMAP[*]}")"
+export OBOT_SERVER_TOOL_REGISTRIES="${TOOL_REGISTRY_REPOS}"
+export OBOT_SERVER_VERSIONS="${OBOT_SERVER_VERSIONS}"
+export TOOLS_VENV_BIN=/obot-tools/venv/bin
```

Comment:
not the prettiest solution -- and breaks outside the context of the container image since all the paths assume are absolute off of the root dir -- but I think it de-spaghettifies the construction of these settings a bit